<?xml version="1.0"?>
<!-- FOR FUTURE USE. NOT SUPPORTED NOW. DO NOT USE IT. -->
<launch>
  <arg name="hostname" default="192.168.0.1" />
    <!-- robot_description and robot_state_publisher hier evtl. einbauen  -->
  <node name="sick_lms_1xxx" pkg="sick_scan" type="sick_generic_caller" respawn="false" output="screen">
    <!-- default values: -->
    <!--
      <param name="scanner_type" type="string" value="sick_lms_1xxx"
      <param name="min_ang" type="double" value="-2.35619449019" />
      <param name="max_ang" type="double" value="2.35619449019" />
      <param name="intensity" type="bool" value="True" />
      <param name="skip" type="int" value="0" />
      <param name="frame_id" type="str" value="laser" />
      <param name="time_offset" type="double" value="-0.001" />
      <param name="publish_datagram" type="bool" value="False" />
      <param name="subscribe_datagram" type="bool" value="false" />
      <param name="device_number" type="int" value="0" />
      <param name="range_min" type="double" value="0.05" />
    -->
    <param name="scanner_type" type="string" value="sick_lms_1xxx"/>
    <param name="range_max" type="double" value="25.0" />
    <param name="hostname" type="string" value="$(arg hostname)" />
    <param name="port" type="string" value="2112" />
    <param name="timelimit" type="int" value="5" />
      <param name="intensity" type="bool" value="true" />
    <param name="intensity_resolution_16bit" type="bool" value="false" />
    <param name="min_ang" type="double" value="-2.391" />
    <param name="max_ang" type="double" value="2.391" />
  </node>
</launch>
